spec/features/proposals_spec.rb

```diff
@@ -43,7 +43,7 @@
 
       click_button 'New'
       click_link "accept_event_#{@event.id}"
-      expect(flash).to eq('Event accepted!')
+      expect(page.has_content?('Event accepted!')).to be true
       expect(page.has_content?('Unconfirmed')).to be true
```

Comment:
or add a new commit to refactor this, but not in the same one that it is used for fixing the flickering tests